### PR TITLE
Typescript installation

### PR DESCRIPTION
**Pull request by MaxLeduc**
- Integrated Typescript and dependencies
- Removed original POC UI and components and did a general cleanup
- Removed `styled-components` package since Shopify are not using it on their end.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,65 +4,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `yarn start`
+### `npm run start-web`
```
Comment (arovillard) [inline README.md:7]: why are we not using `yarn`? the dependency management of yarn is a bit better I think.

**Review by arovillard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MaxLeduc) [reply README.md:7]: I don't have a strong preference either way!

Comment (ruhee) [reply README.md:7]: Now that we know Shopify is using it I think it makes sense to switch over to yarn anyway!

Comment (arovillard) [reply README.md:7]: yarn yarn yarn yarn

**Review by ruhee — APPROVED** [review]
✨   So glad they're using typescript! Runs just fine for me, thanks for setting this up.

**Pull request merged**